simpler/connectors.py

```diff
@@ -136,19 +136,22 @@ def cursor(self):
 					raise ModuleNotFoundError('Missing MS-SQL connector. Install a MS-SQL client and then do `pip install pymssql`.')
 			elif self.engine == 'postgre':
 				try:
-					from psycopg2 import connect
-					from psycopg2.extras import Json
-					from psycopg2.extensions import register_adapter, register_type, new_type, DECIMAL
-					register_adapter(dict, Json)
-					register_type(new_type(
-						DECIMAL.values,
-						'DEC2FLOAT',
-						lambda value, _: float(value) if value is not None else None
-					))
+					from psycopg import connect
+					from psycopg import ClientCursor

```

Comment:
Let's bundle thi line with the previous one (ClientCursor, connect to have it in order)